app/src/main/res/menu/menu_bottom.xml

```diff
@@ -18,6 +18,14 @@
         app:cnb_iconColor="#ffefef"
         app:cnb_textColor="#ffefef"
         />
+    <item
+        android:id="@+id/fridge"
+        android:title="Fridge"
+        android:icon="@drawable/fridge"
+        app:cnb_backgroundColor="#ae0216"
```

Comment:
z kolorami to cię @krecikovsky112  chyba zabije

Reply:
+1 nie wiadomo co to za kolor jest mam nadzieje że pasujący

Reply:
najlepiej byłoby zmienić na nazwę kolor dodane w colors.xml

Reply:
to jest "own_red" ale skopiowałem go z innego pliku, następny PR dam z poprawionym kolorem